cubed/core/array.py

```diff
@@ -105,6 +107,16 @@ def size(self):
         """Number of elements in the array."""
         return reduce(mul, self.shape, 1)
 
+    @property
+    def nbytes(self) -> T_IntOrNaN:
+        """Number of bytes in array"""
+        return self.size * self.dtype.itemsize
+
+    @property
+    def itemsize(self) -> int:
+        """Length of one array element in bytes"""
+        return self.dtype.itemsize
```

Comment:
Both of these are defined on dask arrays. I only strictly needed `nbytes`, but `itemsize` seemed useful too.